docs/src/styles/docs/main.scss

```diff
@@ -1,4 +1,4 @@
-.docs-main p {
+.docs-main p:not([class*='amplify']) {
```

Comment:
We want paragraph tags from MDX content to have a bottom margin, but this was messing up a lot of our components that render p tags.